tests/get_or_create/models.py

```diff
@@ -32,6 +32,14 @@ class Thing(models.Model):
     name = models.CharField(max_length=255)
     tags = models.ManyToManyField(Tag)
 
+    @property
+    def capitalized_name_property(self):
+        return self.name
+
+    @capitalized_name_property.setter
+    def capitalized_name_property(self, val):
+        self.name = val.capitalize()
```

Comment:
@francoisfreitag , I've looked into this and do not understand your suggestion. That `TypeError` is the result of the work in [ticket #25910](https://code.djangoproject.com/ticket/25910). Can you please clarify what you meant?